feature-libs/product-configurator/rulebased/components/exit-button/configurator-exit-button.component.ts

```diff
@@ -5,28 +5,40 @@ import {
   RoutingService,
   WindowRef,
 } from '@spartacus/core';
-import { ConfiguratorRouterExtractorService } from '@spartacus/product-configurator/common';
+import {
+  CommonConfigurator,
+  ConfiguratorRouter,
+  ConfiguratorRouterExtractorService,
+} from '@spartacus/product-configurator/common';
 import { BREAKPOINT, BreakpointService } from '@spartacus/storefront';
 import { Observable } from 'rxjs';
 import { filter, switchMap, take } from 'rxjs/operators';
 import { ConfiguratorCommonsService } from '../../core/facade/configurator-commons.service';
 import { Configurator } from '../../core/model/configurator.model';
+import { Location } from '@angular/common';
 
 @Component({
   selector: 'cx-configurator-exit-button',
   templateUrl: './configurator-exit-button.component.html',
 })
 export class ConfiguratorExitButtonComponent {
   product$: Observable<Product>;
+  routerData$: Observable<ConfiguratorRouter.Data> =
+    this.configRouterExtractorService.extractRouterData();
 
   constructor(
     protected productService: ProductService,
     protected routingService: RoutingService,
     protected configRouterExtractorService: ConfiguratorRouterExtractorService,
     protected configuratorCommonsService: ConfiguratorCommonsService,
     protected breakpointService: BreakpointService,
-    protected windowRef: WindowRef
+    protected windowRef: WindowRef,
+    protected location: Location
```

Comment:
Please add an issue that covers this breaking change.  See e.g. #14352

Reply:
An issue has been created. Please check if it is correctly done.